docs/content/docs/pub-sub-implementations.md

```diff
@@ -146,12 +146,10 @@ marshaler := kafka.NewWithPartitioningMarshaler(func(topic string, msg *message.
 
 ### HTTP
 
-At this moment only HTTP subscriber is available. There is an issue for a [HTTP publisher](https://github.com/ThreeDotsLabs/watermill/issues/17).
-
-HTTP subscriber allows us to send messages received by HTTP request (for example - webhooks).
+The HTTP subscriber listens to HTTP requests (for example - webhooks) and outputs them as messages.
 You can then post them to any Publisher. Here is an example with [sending HTTP messages to Kafka](https://github.com/ThreeDotsLabs/watermill/blob/master/_examples/http-to-kafka/main.go).
 
-When implemented, HTTP publisher can be used as webhooks sender.
+The HTTP publisher sends HTTP requests as specified in its configuration. Here is an example with [transforming Kafka messages into HTTP webhook requests](https://github.com/ThreeDotsLabs/watermill/tree/http-publisher/_examples/kafka-to-http).
```

Comment:
link will be broken after merge and branch remove

Reply:
988ec6531abbd023c5fcba55b153e3c8ba953f13